SYCL/Regression/cache_test.cpp

```diff
@@ -91,8 +93,8 @@ int queryFromNativeHandle(std::vector<cl::sycl::platform> *platform_list,
   zeDeviceGet(l0_drivers[0], &l0_device_count, l0_devices.data());
 
   // Create the platform and device objects using the native handle.
-  auto plt = cl::sycl::level_zero::make<cl::sycl::platform>(l0_drivers[0]);
-  auto dev = cl::sycl::level_zero::make<cl::sycl::device>(plt, l0_devices[0]);
+  auto plt = level_zero::make<cl::sycl::platform>(l0_drivers[0]);
```

Comment:
non-blocking comment: "cl::" here could be removed. I.e. "sycl::platform" instead of "cl::sycl::platform"